eventstore/eventstore.go

```diff
@@ -106,7 +106,15 @@ func migrate(store EventStore) error {
 	})
 }
 
-func getEventsToMigate(db *bolt.DB) ([]Event, []EventTimes, error) {
+// copyBytes gets used to copy the keys and value from boltdb to prevent
+// 'unexpected fault address' errors.
+func copyBytes(b []byte) []byte {
+	out := make([]byte, len(b))
+	copy(out, b)
+	return out
+}
```

Comment:
I've just remembered: another way to copy a slice is to use append like this: https://play.golang.org/p/F1Go3oH5ZhM

You almost don't need `copyBytes` then.